libafl_libfuzzer/runtime/Cargo.toml.template

```diff
@@ -4,16 +4,19 @@ version = "0.15.2"
 edition = "2024"
 publish = false
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [features]
-default = ["fork"]
+default = []
 ## Enables forking mode for the LibAFL launcher (instead of starting new processes)
 fork = ["libafl/fork"]
 track_hit_feedbacks = [
   "libafl/track_hit_feedbacks",
   "libafl_targets/track_hit_feedbacks",
 ]
+tui_monitor = ["libafl/tui_monitor"]
+
+[target.'cfg(not(windows))'.features]
+## Enable the `fork` feature on non-windows platforms
+default = ["fork", "tui_monitor"]
```

Comment:
don't think we need tui_monitor on non-win

Reply:
I have to put in a few libs anyway to get this to work so I'll just revert this part of the changes and make tui_monitor work.

Reply:
Don't fully get the tui monitor thing - it's just not working on Win yet?